srcStatic/ModuleLoader.cpp

```diff
@@ -29,13 +28,9 @@ void ModuleLoader::RegisterBuiltInModules()
 {
 	const std::string sectionName("BuiltInModules");
 
-	if (IsModuleRequested(sectionName, "IPDropDown")) {
+	if (IsModuleRequested(sectionName, "IPDropDown", true)) {
```

Comment:
This boolean argument added to IsModuleRequested represents the default return if the iniFile does not contain an entry for a built in module. Implementation before 3.1.0 would not load a built in module if it was not present in the ini file. 3.1.0 adds the ability to set a default for each module whether it should load or not load if not present in the ini file. 

This is causing some of the unit test failures. Several unit tests had assumed IP Drop Down would not load if not present in a blank ini file which is not true in 3.1.0. Should be easy to account for in the unit tests. 

If a change were made to improve readability of this source code line, it might be easier to use an enum class to be more descriptive of what the boolean meant than just passing a boolean. I would probably prefer this to multiple functions in this case but probably more of a personal preference. I think there is a desire to remove the concept of built in modules in the future though.